test/unit/vdom/events.js

```diff
@@ -108,16 +85,33 @@ describe('vdom/events (on*)', () => {
 
   it('should not fail for listeners that are not functions', done => {
     const myel = new (element().skate({
-      render () {
-        vdom.element('div', { 'on-test': null });
+      props: {
+        handler: {}
+      },
+      render (elem) {
+        return h('div', { onTest: elem.handler });
       }
     }))();
     fixture(myel);
+    let div
     afterMutations(
-      () => {}, // .render()
-      () => {
-        emit(myel.shadowRoot.firstChild, 'test');
-      },
+      () => (div = myel.shadowRoot.firstChild),
```

Comment:
I added several more assertions here since we rely on `null` and `undefined` explicitly in the event binding logic and we need to make sure that if an event is set to one of those that it doesn't screw it up. Also passed falsy / truthy values to ensure that we only allow functions.